### PR TITLE
Minor updates to avoid errors and speed up state root matching

### PR DESCRIPTION
**Pull request by smartcontracts**

### CODE REVIEW

File: src/services/fraud-prover.service.ts [hunk 1]
```diff
@@ -306,7 +306,14 @@ export class FraudProverService extends BaseService<FraudProverOptions> {
...
               err
                 .toString()
-                .includes('Function must be called during the correct phase.')
+                .includes(
+                  'Function must be called during the correct phase.'
+                ) ||
+              err
+                .toString()
+                .includes(
+                  '46756e6374696f6e206d7573742062652063616c6c656420647572696e672074686520636f72726563742070686173652e'
```
Comment (smartcontracts) [inline src/services/fraud-prover.service.ts:315]: This is the byte representation of `Function must be called during the correct phase`. Some providers only return it in bytes :-(

**Review by karlfloersch — DISMISSED** [review]
LGTM!

**Review by K-Ho — APPROVED** [review]
lgtm!

File: src/services/fraud-prover.service.ts (hunk 1 above)
Comment (tynes) [reply src/services/fraud-prover.service.ts:315]: Maybe there should be a helper fn that checks for both possibilities and then can be used in each place that the revert message is used, can use `Buffer.from(message, 'ascii').toString('hex')`

**Pull request merged**